src/index.js

```diff
@@ -47,7 +48,7 @@ export default function supernova(env) {
       const [opts] = useState(options);
       const selectionsAPI = useSelections();
       const constraints = useConstraints();
-      const [selections] = useState({ api: selectionsAPI, setState: setSelectionState, linked: false });
+      const [selections] = useState({ api: selectionsAPI, setState: setSelectionState, linked: false, zoom: {} });
```

Comment:
Let's go for selectionsAndZoom if we cannot come up with anything better for now